.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,4 @@
-<!--- markdownlint-disable MD041 -->
+<!--- markdownlint-disable -->
 # What does this PR do?
```

Comment:
Should we add a main title for the template (`New Pull Request:`) and make other headers as sub-headers (`##`)?

Reply:
I don't think it really makes a lot of sense to do so. This is the same change we have made in the other repos. It would be unnecessary to have a title in there for each PR, that'll just be extra noise

Reply:
Could the `What does this PR do?` be the main title?